react/ProductSummaryImage.tsx

```diff
@@ -153,15 +154,18 @@ function CollectionWrapper({
   showCollections,
   productClusters,
   children,
+  classes,
```

Comment:
You should type this prop like [this](https://github.com/vtex-apps/product-summary/blob/1d908de85f310541f058e0449a2aa553b8a1f783/react/ProductSummarySKUName.tsx#L11).